### DISCUSSION TITLE
Reader Mode

### DISCUSSION THREAD

**Discussion by pratik-chakravorty** [dailydotdev/daily#530] [Feature Requests]
Instead of having to go to the link is it possible to fetch the content of the link and then display them within the app like some rss readers allow?

**Comment by nimrodkra** [comment]
Hi @pratik-chakravorty currently we don't plan to embed the content itself in the app due to various considerations. Primarily we are considering the ethical aspects of it. However, to make things easier we build a new article modal that easily allows you to skim through articles and read their TLDR before deciding to read the full article. https://app.daily.dev/posts/RVQ3Q47Gc

Try clicking on the comment button on a card to open the modal and then use the keyboard arrows (of the J/K keys) to go to the next/previous article :)
